File: ViewDemo/app/src/main/java/com/saltwater/viewdemo/MainActivity.java
package com.saltwater.viewdemo;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        findViewById(R.id.btn_base).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(MainActivity.this,BasisActivity.class));
            }
        });

        findViewById(R.id.btn_event).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(MainActivity.this,EventActivity.class));
            }
        });
        findViewById(R.id.btn_scroll).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(MainActivity.this,ScrollActivity.class));
            }
        });
    }

}

File: View/WebViewDemo/app/src/main/java/com/saltwater/zhaoshanbangka_kotlin/MainActivity.kt
package com.saltwater.zhaoshanbangka_kotlin

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btn_start.setOnClickListener{
            val intent = Intent()
            intent.putExtra("url",edit_url.text)
            intent.putExtra("phone",edit_phone.text)
            intent.putExtra("plate",edit_plate.text)
            intent.setClass(this,WebViewActivity::class.java)
            startActivity(intent)
        }
    }
}
